AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -122,8 +122,8 @@ - (int)executeQueryUsingBlock:(MSDBStorageQueryBlock)callback {
 - (BOOL)dropTable:(NSString *)tableName {
   return [self executeQueryUsingBlock:^int(void *db) {
            if ([MSDBStorage tableExists:tableName inOpenedDatabase:db]) {
-             NSString *deleteQuery = [NSString stringWithFormat:@"DROP TABLE \"%@\";", tableName];
```

Comment:
I guess we should restore `;`

Reply:
SQL syntax require delimiter in the end of query for DB to start processing query.